src/api/xml/request-util.ts

```diff
@@ -0,0 +1,104 @@
+import { GinisConfig } from '../../ginis'
+import { parseString } from 'xml2js'
+
+export type XmlRequestInfo = {
+  name: string
+  namespace: string
+  xrgNamespace: string
+  paramsBody: any
+  paramOrder: readonly string[]
+}
+
+export function createXmlRequestConfig(requestName: string, requestNamespace: string) {
+  return {
+    headers: createXmlRequestHeader(requestName, requestNamespace),
+  }
+}
+
+export function createXmlRequestHeader(requestName: string, requestNamespace: string) {
```

Comment:
this seems like needless indirection, would keep it all inside `createXmlRequestConfig` fn